### ISSUE TITLE
react-annotation note settings not honored with marginalia

### ISSUE THREAD

**Issue by susielu** [emeeks/semiotic#194]
When I have marginalia settings: 
```js 
  annotationSettings: {
    layout: { type: "marginalia", orient: ["right"] }
  }
```
and settings in my annotations' notes:
```js
            note: {
              label: match.title_desc.split(": Season")[0],
              orientation: "leftRight",
              align: "middle"
            },
```
the note orientation and alignment aren't honored.

**Comment by emeeks** [comment]
Currently marginalia tries to compute smart orientation and alignment settings (the main use case being that the left and right positioned labels should be `leftRight` and `left` or `right` accordingly) but in the next version it will let you override them.

**Comment by emeeks** [comment]
Available in 1.9.3
